Suggest adding a Radar link to internal contributors when running git-webkit pr

alancoon opened the pull request:
#### b644cfdda4fc18287487baaded4e288b3bc80364
<pre>
Suggest adding a Radar link to internal contributors when running git-webkit pr
<a href="https://bugs.webkit.org/show_bug.cgi?id=242914">https://bugs.webkit.org/show_bug.cgi?id=242914</a>
rdar://97276993

Reviewed by Jonathan Bedard.

* Tools/Scripts/hooks/prepare-commit-msg: add logic to suggest internal contributors
add a Radar link to their commits
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/squash.py:
(Squash.squash_commit): drive-by typo fix

Canonical link: <a href="https://commits.webkit.org/252672@main">https://commits.webkit.org/252672@main</a>
</pre>

JonWBedard commented on Tools/Scripts/hooks/prepare-commit-msg:
```diff
@@ -10,6 +10,29 @@ SCRIPTS = os.path.dirname(os.path.dirname(LOCATION))
+    """Determines what bug URLs to fill or suggest in a WebKit commit message."""
+    env = os.environ
+    need_the_bug_url = 'Need the bug URL (OOPS!).'
+    need_the_radar = 'Heavily recommended: include a Radar link with no title (OOPS!).'
```
Would just change the to `Include a Radar link (OOPS!)`

youssefsoli replied:
This would be a bit awkward to deal with until automatic radar importing is enabled w.r.t bugzilla right?

JonWBedard replied:
We already have some automatic radar importing, in that if you give `git-webkit` a bugzilla which is cloned into radar, we'll pick up the radar too. @alancoon and I do need to make this particular behavior cover a few more cases.

alancoon committed:
Suggest adding a Radar link to internal contributors when running git-webkit pr
https://bugs.webkit.org/show_bug.cgi?id=242914
rdar://97276993

Reviewed by Jonathan Bedard.

* Tools/Scripts/hooks/prepare-commit-msg: add logic to suggest internal contributors
add a Radar link to their commits
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/squash.py:
(Squash.squash_commit): drive-by typo fix

Canonical link: https://commits.webkit.org/252672@main
